src/CurrenciesSpecification.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Money;
+
+use Money\Currencies\Specification;
+
+/**
+ * Specifies information about a currency.
+ *
+ * @author Frederik Bosch <f.bosch@genkgo.nl>
+ */
+interface CurrenciesSpecification
+{
+    /**
+     * @param Currency $currency
+     *
+     * @return Specification
```

Comment:
And I would rename `Specification` -> `CurrecySpecification`. I think that it will be clearer.